highfidelity/hifi
# Added support for Google Blocks .fbx formats

Test plan:

1. Open up Blocks app from the marketplace
2. Find a model that lists "fbx" as a file type when you click the "Download" button, then click that
3. The model should appear in front of you
4. Try with the same model but choose "obj"; the model should still appear in front of you
